src/View/StringTemplate.php

```diff
@@ -29,7 +29,7 @@ class StringTemplate
 {
 
     use InstanceConfigTrait {
-        config as get;
+        getConfig as get;
```

Comment:
We did the same for Log and alike afaik. Writing here was never supposed to be done for "get".